### ISSUE TITLE
Timeline response filled with None; Unexpected error during Timeline download

### ISSUE THREAD

**Issue by TreeForTwo** [prof79/fansly-downloader-ng#35]
## Bug Description
Cannot download creator media. Program errors during Timeline download, with all response fields being `[None]`.

Single post mode works as intended. Normal and Timeline fails.

## Expected behavior
The program downloads all creator media from their timeline

## Actual behavior
The program fails with the following error, and continuously retries with the same failure

```
 DEBUG | 21:55 || Timeline object: {'posts': [None], 'accountMediaBundles': [None], 'accountMedia': [None], 'accounts': [None], 'tips': [None], 'timelineReadPermissionFlags': [None], 'accountTimelineReadPermissionFlags': {'flags': 0, 'metadata': '{}'}}
 [36]ERROR | 21:55 || Unexpected error during Timeline download: 
Traceback (most recent call last):
  File "/home/treefortwo/Downloads/fansly-downloader-ng-0.8.19/download/timeline.py", line 67, in download_timeline
    all_media_ids = get_unique_media_ids(timeline)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/treefortwo/Downloads/fansly-downloader-ng-0.8.19/download/common.py", line 45, in get_unique_media_ids
    account_media_ids = [
                        ^
  File "/home/treefortwo/Downloads/fansly-downloader-ng-0.8.19/download/common.py", line 46, in <listcomp>
    media['id']
    ~~~~~^^^^^^
TypeError: 'NoneType' object is not subscriptable

```

## Environment Information
 - Operating System: Linux, Fedora 39, Python3.11 in a venv
 - Fansly Downloader Version: 0.8.19
 - Fansly Downloader Type: Python code
 - Specific creators name: @creamyspot

## User Research
I have done the following:
- [ x] Looked through [old GitHub issues](https://github.com/prof79/fansly-downloader-ng/issues?&q=is%3Aissue)
- [ x] Read the [config.ini options](https://github.com/prof79/fansly-downloader-ng/wiki/Fansly-Downloader-NG-Settings#explanation-of-configini)
- [x ] Searched the Web (like [DuckDuckGo](https://duckduckgo.com))

**Comment by ZincStoat** [comment]
Duplicate of #34

**Comment by prof79** [comment]
Closed as duplicate of #34
